src/kaocha/load.clj

```diff
@@ -21,8 +21,8 @@
              (filter (partial ns-match? ns-patterns)))
             test-paths))
 
-(defn load-test-namespaces [testable ns-testable-fn & [platform]]
-  (let [test-paths  (:kaocha/test-paths testable)
+(defn load-namespaces [testable paths-key ns-testable-fn & [platform]]
```

Comment:
This function is also used in other libraries like `kaocha-midje`, so we should keep this signature intact. Instead I think there's a simple refactoring where `load-test-namespaces` pulls out `:kaocha/test-paths`, and then calls `load-namespaces` with the paths.

``` clojure
(defn load-namespaces [testable paths ns-testable-fn & [platform]]
  ,,,)

(defn load-test-namespaces [testable ns-testable-fn & [platform]]
  (load-namespaces testable (:kaocha/test-paths testable) ns-testable-fn platform))
```